docs/content/examples/tutorials/basic-installation.md

```diff
@@ -56,11 +58,6 @@ services:
       - ./docker-data/dms/config/:/tmp/docker-mailserver/
       - /etc/localtime:/etc/localtime:ro
     environment:
-      - ENABLE_SPAMASSASSIN=1
-      - SPAMASSASSIN_SPAM_TO_INBOX=1
-      - ENABLE_CLAMAV=1
-      - ENABLE_FAIL2BAN=1
```

Comment:
The LDAP example should be as simple as possible, so I removed these ENVs. I am actually thinking about removing the "Basic Installation" example above completely later, because it essentially adds no more information that you cannot already find on the "Introduction" and "Usage" pages.